### PR TITLE
feat: Add CDN example app

### PR DESCRIPTION
**Pull request by cola119**
Page: https://line.github.io/liff-mock/examples/cdn-example/

This PR added super simple example app for CDN usage.
The target brunch of GitHub pages is now `add-cdn-example` but I'll change it to `main` after this PR is landed.

### CODE REVIEW

File: examples/cdn-example/index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>LIFF Mock example</title>
+    <script
+      charset="utf-8"
+      src="https://static.line-scdn.net/liff/edge/versions/2.19.0/sdk.js"
```
Comment (LeeChSien) [inline examples/cdn-example/index.html:10]: Why don't we use latest version?

Comment (cola119) [reply examples/cdn-example/index.html:10]: Currently liff-mock@1.0.2 supports LIFF SDK v2.19.0 or earlier.
https://github.com/line/liff-mock/blob/f7d6727338cbb7a866ec3c8d418819b665d4bd46/package.json#L34-L35

I'll add the support for the latest SDK in a later PR but this PR is enough to support 2.19.0.

**Review by LeeChSien — APPROVED** [review]
LGTM

**Review by LeeChSien — COMMENTED** [review]
Sorry, waif for a minute

File: examples/cdn-example/index.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  } catch (e) {
+    console.error('liff.init error', e);
+  }
+};
+
+const login = async () => {
+  try {
+    await liff.login();
+  } catch (e) {
+    console.log('liff.login done');
```
Comment (LeeChSien) [inline examples/cdn-example/index.js:27]: Move to try block?

Comment (cola119) [reply examples/cdn-example/index.js:27]: Good Catch! DONE 92c17971259fd98740451878eb7e83cfa163d08b

**Review by so99ynoodles — APPROVED** [review]
LGTM

**Pull request merged**